### ISSUE TITLE
Pasting Hyperlinks in Writer/Word doesnt link hyperlinks

### ISSUE THREAD

**Issue by b90g** [ONLYOFFICE/DocumentServer#961]
**Do you want to request a *feature* or report a *bug*?**
Bug
**What is the current behavior?**
Hyperlinks dont get translated into clickable hyperlinks
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Paste a Hyperlink into a docx-file.
Link is just Fext
**What is the expected behavior?**
Link gets formed into a clickable Hyperlink with blue text and underlined...
**Did this work in previous versions of DocumentServer?**
Users told me it worked before i am not sure.
**DocumentServer version:**
5.4.2 (?)
**Operating System:**
Ubuntu 18.04 on server. - Various on client. (Windows and Linux)
**Browser version:**
Firefox 79, but others are probably as well.

**Comment by ShockwaveNN** [comment]
Hi, could you update to DocumentServer latest version and check again?

Also by any chance - are you using Nextcloud's Community DocumentServer?

**Comment by jdaviescoates** [comment]
> Hi, could you update to DocumentServer latest version and check again?

This still seems to be the case for me. I'm using ONLYOFFICE Document Server 5.6.5

> Also by any chance - are you using Nextcloud's Community DocumentServer?

I am using Nextcloud (19.0.4), but it's connected to my ONLYOFFICE Document Server 5.6.5 using the [ONLYOFFICE Nextcloud app](https://apps.nextcloud.com/apps/onlyoffice) 6.0.0 (although I've just seen there is an update to 6.0.2 so will run that and report back...)

**Comment by ShockwaveNN** [comment]
Yeah, DocumentServer 5.6.5 is the latest version, but I'm not sure this is same problem as from orignal author, who didn't answer any question.

@jdaviescoates Could you describe your scenario in more details?

**Comment by jdaviescoates** [comment]
The same as the original post. When pasting links into a document that don't automatically become hyperlinks. You have to choose hyperlink from the right click context menu and then paste it again into that box.

**Comment by Rita-Bubnova** [comment]
Hello, @jdaviescoates.
I can confirm, this is know bug - issue 28796 in our private issue tracker.

**Comment by b90g** [comment]
i wasnt able so far to update the document server, sorry i didnt answer.

**Comment by b90g** [comment]
I tried with 6.0.0, firefox is now 84.

**Comment by jdaviescoates** [comment]
Just checked with ONLYOFFICE Docs 6.1.0 and both Firefox 84.0.2 and Chromium 87.0.4280.141 (Official Build) snap (64-bit) and still the same behaviour, i.e. pasted links do not automatically become links.

**Comment by christophebe** [comment]
The same for me : Only Office server 6.3 and Nextcloud 21.0.1


> Hello, @jdaviescoates.
> I can confirm, this is know bug - issue 28796 in our private issue tracker.

Is it still open ? 

Thanks

**Comment by Rita-Bubnova** [comment]
> Is it still open ?

The problem has now been partially fixed.
Thank you for bringing the issue to our attention.
